src/oaklib/implementations/simpleobo/simple_obo_implementation.py

```diff
@@ -795,14 +795,17 @@ def apply_patch(
                 raise ValueError(f"Failed to find synonym {patch.old_value} for {t.id}")
             modified_entities.append(patch.about_node)
         elif isinstance(patch, kgcl.NewTextDefinition):
+            v = f'"{patch.new_value}"'
```

Comment:
in contrast to pronto this is fine, as with this adapter you are working closer to the level of the ascii...

but where is the trailing `[]`s?

Reply:
It's a `NewTextDefinition`, so if there is a `[]` provided it will be added as long as it is within the double quotes. Or am I misunderstanding the question?